### PR TITLE
GitHub info

### PR DESCRIPTION
**Pull request by mlachkar**

### CODE REVIEW

File: modules/server/src/main/scala/scaladex/server/service/PublishProcess.scala [hunk 1]
```diff
@@ -34,7 +37,7 @@ class PublishProcess(
     database: WebDatabase,
     pomsReader: PomsReader,
     env: Env
-)(implicit ec: ExecutionContext)
+)(implicit ec: ExecutionContext, system: ActorSystem)
```
Comment (adpi2) [inline modules/server/src/main/scala/scaladex/server/service/PublishProcess.scala:40]: Why do we need an `ActorSystem` here?

File: modules/core/shared/src/main/scala/scaladex/core/service/WebDatabase.scala [hunk 2]
```diff
@@ -7,11 +7,13 @@ import scala.concurrent.Future
...
 import scaladex.core.model.Artifact
 import scaladex.core.model.ArtifactDependency
+import scaladex.core.model.GithubInfo
+import scaladex.core.model.GithubResponse
 import scaladex.core.model.Project
 import scaladex.core.model.UserState
 
 trait WebDatabase {
-  def insertArtifact(artifact: Artifact, dependencies: Seq[ArtifactDependency], time: Instant): Future[Unit]
+  def insertArtifact(artifact: Artifact, dependencies: Seq[ArtifactDependency], time: Instant): Future[Boolean]
```
Comment (adpi2) [inline modules/core/shared/src/main/scala/scaladex/core/service/WebDatabase.scala:16]: Can you add a comment to explain what this boolean means?

File: modules/infra/src/main/scala/scaladex/infra/migrations/V7_2__edit_platform_and_language.scala [hunk 3]
```diff
@@ -24,12 +24,10 @@ class V7_2__edit_platform_and_language extends BaseJavaMigration with ScaladexBa
       (for {
         oldArtifacts <- run(xa)(selectArtifact.to[Seq])
         groupedArtifacts = oldArtifacts.grouped(10000).toSeq
-        numberOfArtifactsUpdated <- groupedArtifacts
+        _ <- groupedArtifacts
           .map(artifacts => run(xa)(updatePlatformAndLanguage.updateMany(artifacts.map(_.update))))
           .sequence
-        _ = logger.info(s"Updated ${numberOfArtifactsUpdated.sum} artifacts")
```
Comment (adpi2) [inline modules/infra/src/main/scala/scaladex/infra/migrations/V7_2__edit_platform_and_language.scala:30]: So we can change the code of migration files that were already executed?

**Review by adpi2 — COMMENTED** [review]
I left some minor comments but that looks great!

File: modules/server/src/main/scala/scaladex/server/service/PublishProcess.scala (hunk 1 above)
Comment (mlachkar) [reply modules/server/src/main/scala/scaladex/server/service/PublishProcess.scala:40]: for githubClient

File: modules/core/shared/src/main/scala/scaladex/core/service/WebDatabase.scala (hunk 2 above)
Comment (mlachkar) [reply modules/core/shared/src/main/scala/scaladex/core/service/WebDatabase.scala:16]: Not my favourite trick.. I will add a comment

File: modules/infra/src/main/scala/scaladex/infra/migrations/V7_2__edit_platform_and_language.scala (hunk 3 above)
Comment (mlachkar) [reply modules/infra/src/main/scala/scaladex/infra/migrations/V7_2__edit_platform_and_language.scala:30]: yes!

Comment (mlachkar) [reply modules/infra/src/main/scala/scaladex/infra/migrations/V7_2__edit_platform_and_language.scala:30]: I mean at least for the one with code!

Comment (adpi2) [reply modules/infra/src/main/scala/scaladex/infra/migrations/V7_2__edit_platform_and_language.scala:30]: That's a good news! :slightly_smiling_face:

**Pull request merged**